packages/manager/modules/web-paas/src/details/service/service.html

```diff
@@ -93,7 +93,8 @@
                     <oui-tile-description>
                         <p data-ng-if="$ctrl.project.addonStorageCount() > 0">
                             <span
-                                data-ng-bind="$ctrl.project.getTotalStorage()"
+                                data-translate="web_paas_service_project_storage_included"
+                                data-translate-values="{storage: $ctrl.project.getTotalStorage() - ($ctrl.project.addonStorageCount() * $ctrl.STORAGE_MULTIPLE)}"
```

Comment:
This logic should be done into the controller

Reply:
Has been donen